test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -186,7 +186,9 @@ export default function Main(props: WalletConnectionProps) {
             },
         });
 
-        fetch(target.value)
+        const schemaURL = target.value.replace('http://', 'https://');
```

Comment:
It would be good if we enabled as many test scenarios to be executed via the front end as possible. Replacing the prefix `http` with `https` allows our hosted front end to fetch and display the input fields at the front end, so that our testers can continue their testing flows. This change allows our testers to check out how the browser wallet reacts when using a schema over the `http` protocol. It would mean for testing you need to upload your schema at the hosting page that can serve the schema via `https` as well as `http`.


Not replacing it, would mean you need to run the front end locally to test `http` schema because the following error would appear on our hosted frontend:
```
Main.tsx:234 Mixed Content: The page at 'https://web3id-issuer-frontend.testnet.concordium.com/' was loaded over HTTPS, but requested an insecure resource 'http://...../'.  This request has been blocked; the content must be served over HTTPS.
```

https://concordium.atlassian.net/browse/CBW-1272

Reply:
Right, but I don't think we should be doing that this way.

You can (at least in chrome) allow this mixed content.

You click on the shield icon

![image](https://github.com/Concordium/concordium-web3id/assets/105212/67df61d4-0a60-4dc7-b9da-24e030739577)

Then site settings and allow insecure content

![image](https://github.com/Concordium/concordium-web3id/assets/105212/d317d546-8536-4a46-aea5-59721efa4cc7)

Reply:
That is set for the domain, so they only need to do it once.